tensorflow_lite_support/python/task/audio/core/tensor_audio.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""TensorAudio class."""
+
+import numpy as np
+
+from tensorflow_lite_support.python.task.audio.core.pybinds import audio_utils
+from tensorflow_lite_support.python.task.audio.core.pybinds import audio_buffer
+
+
+class TensorAudio(object):
+  """A wrapper class to store the input audio."""
+
+  def __init__(self,
+               audio_format: audio_buffer.AudioFormat,
+               sample_count: int,
+               audio_data: audio_utils.AudioData = None
+               ) -> None:
+    """Initializes the `TensorAudio` object."""
+    self._format = audio_format
+    self._sample_count = sample_count
+    self._buffer = np.zeros([self._sample_count, self._format.channels])
+    self._data = audio_data
+    self.clear()
+
+  def clear(self):
+    """Clear the internal buffer and fill it with zeros."""
+    self._buffer.fill(0)
+
+  def load_from_file(self, file_name: str) -> audio_utils.AudioData:
```

Comment:
This should be a class method:
```
@classmethod
def from_wav_file(cls, file_name: str) -> "TensorAudio":
```

The method should create a AudioBuffer instance from the WAV file and store it internally in the TensorAudio.

Reply:
@khanhlvg Since I'm using a method to create the `TensorAudio` object from `AudioClassifier`'s `required_input_buffer_size` and 'required_audio_format' properties [here](https://github.com/khanhlvg/tflite-support/blob/audio-classifier-python/tensorflow_lite_support/python/task/audio/audio_classifier.py#L92) prior to loading the WAV file, I figured it would make sense to make it a non-class method.